packages/autocomplete-js/src/createAutocompleteDom.ts

```diff
@@ -53,30 +48,35 @@ export function createAutocompleteDom<TItem extends BaseItem>({
     props: autocomplete.getRootProps({}),
     ...autocompleteScopeApi,
   });
-  const root = Element('div', { class: classNames.root, ...rootProps });
-  const touchOverlay = Element('div', { class: classNames.touchOverlay });
+  const root = createDomElement('div', {
```

Comment:
See "Next steps" section in post about using pragmas for the search box in the future too. We might not need to manipulate the DOM ourselves for these UI elements too.